charts/k8ssandra-cluster/values.yaml

```diff
@@ -6,6 +6,8 @@ name: k8ssandra
 clusterName: k8ssandra
 datacenterName: dc1
 size: 1
+allowMultipleNodesPerWorker: false
```

Comment:
If we have `allowMultipleNodesPerWorker: true`, should there be a validation check for resources being defined?

Reply:
At this time the validation of the CassandraDatacenter crd gives a pretty clear error message when you fail to set the resources:
`Error: admission webhook "cassandradatacenter-webhook.cassandra.datastax.com" denied the request: CassandraDatacenter write rejected, attempted to use multiple nodes per worker without cpu and memory requests and limits`

I could add some conditional statements but it would make the template bigger, and the error message less clear I think. Maybe a comment in the default values.yaml?

Reply:
The error message from the webhook is great. The problem is that the chart install still happens and other objects still get created. The user would have to `helm uninstall` before trying the install again. We could use the `required` template function to make sure `resources` is defined. It would fail fast before anything gets deployed.

Reply:
Ah yeah good point!